src/styles/themes/default.ts

```diff
@@ -19,7 +21,6 @@ const darkTheme = {
     iconColorfulBackground: `${colors.ivory}cc`,
     textSupporting: colors.darkSupportingText,
     text: colors.darkPrimaryText,
-    textColorfulBackground: colors.ivory,
```

Comment:
Is this unused?

Reply:
Not sure why this was removed tbh. Gonna add it back